packages/headless/package.json

```diff
@@ -31,7 +31,7 @@
     "test:watch": "jest --watch --colors --no-cache --silent=false",
     "test:unit": "jest --testPathIgnorePatterns=src/integration-tests",
     "test:integration": "jest --testPathPattern=src/integration-tests",
-    "npm:publish": "npm publish --access public",
+    "npm:publish": "node ../../scripts/deploy/is-published.js . || npm publish --access public",
```

Comment:
Since we are introducing a script, why not execute the command `npm publish --access public` inside the script too?

Alternatively, wouldn't the `--no-bail` parameter remove the need for a script?

Reply:
`--no-bail` seems to omit errors.

Reply:
ok, I thought`--verbose` might have changed the behaviour of `--no-bail` to not omit errors 👍